src/main/java/seedu/address/logic/commands/RenameTagCommand.java

```diff
@@ -0,0 +1,105 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+import seedu.address.model.tag.Tag;
+
+/**
+ * Renames a predefined tag.
+ */
+public class RenameTagCommand extends Command {
+    public static final String COMMAND_WORD = "renametag";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Renames an existing tag.\n"
+            + "Example: " + COMMAND_WORD + " t/Bride's Friend t/Friend";
```

Comment:
I think we should standardise all examples to use t/bride's side t/friends, meaning no capitals and bride's side instead of friend, and friends instead of friend